src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -80,34 +80,46 @@
     {%  if draft.formaliseRepaymentPlan.option.value === FormaliseRepaymentPlanOption.SIGN_SETTLEMENT_AGREEMENT.value %}
       {{ tableStart('settlementAgreement', 'Settlement agreement') }}
 
-      {% if draft.courtOfferedPaymentIntention.paymentOption === PaymentType.BY_SPECIFIED_DATE.value %}
-        {% set amountToPay = 'the full amount' %}
-        {% if claim.response.responseType == ResponseType.PART_ADMISSION.value %}
-          {% set amountToPay = claim.response.amount | numeral %}
-        {% endif %}
-
-        {{ row(label = 'The offer',
+      {% if draft.courtOfferedPaymentIntention.paymentOption === PaymentType.IMMEDIATELY.value %}
```

Comment:
To sign a settlement agreement, we are always using `courtOfferedPaymentIntention`, which does not cover the scenario where claimant accepts the defendant payment plan.

I think in this case where claimant accepts the defendant payment plan, the settlement should use  payment intention from defendant response object and NOT `courtOfferedPaymentIntention` as it would be `undefined`

Reply:
very good point...done now